### PR TITLE
Bump preview command

### PR DESCRIPTION
**Pull request by scharrier**
This is the implementation of the first command, the `preview` one. This one creates a temporary documentation preview (which expires 30mn later). It was the simplest one to implement, as there is not authentication for creating previews.

Common command code will be extracted in the next PR, when we have more than one command.

### CODE REVIEW

File: spec/bump/commands/preview_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+require "spec_helper"
+
+describe Bump::CLI::Commands::Preview do
+  it 'calls the server and display the link on success' do
+    stub_and_return('previews/post_success.http')
+
+    expect do
+      new_command.call(file: __FILE__, format: 'yaml')
```
Comment (mehlah) [inline spec/bump/commands/preview_spec.rb]: I don't get the `__FILE__` here 🤔 Can you pls clarify?

File: spec/bump/commands/preview_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    end.to output(/Preview created/).to_stdout
+
+    expect(WebMock).to have_requested(:post,'https://bump.sh/api/v1/previews')
```
Comment (mehlah) [inline spec/bump/commands/preview_spec.rb]: Don't you want to expect the request body is correct too? (the format and the file)

File: spec/bump/commands/preview_spec.rb [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        new_command.call(file: __FILE__, format: 'yaml')
+      rescue SystemExit; end
+    end.to output(/Something bad happened./).to_stderr
+
+    expect(WebMock).to have_requested(:post,'https://bump.sh/api/v1/previews')
+  end
+
+  private
+
+  def stub_and_return(path)
```
Comment (mehlah) [inline spec/bump/commands/preview_spec.rb]: This method names confused me. I was wondering what's stubbed and looking for the `return` but actually it's just a stub for an API call. Naming it `stub_bump_api_create_preview` could be better?

File: spec/bump/commands/preview_spec.rb (hunk 1 above)
Comment (scharrier) [reply spec/bump/commands/preview_spec.rb]: Oh, I think I was lazy here: I needed a real file to make `open` work. I'd better stub the `open` method.

File: spec/bump/commands/preview_spec.rb (hunk 2 above)
Comment (scharrier) [reply spec/bump/commands/preview_spec.rb]: Right !

File: spec/bump/commands/preview_spec.rb (hunk 3 above)
Comment (scharrier) [reply spec/bump/commands/preview_spec.rb]: :+1:

**Pull request merged**